src/plugins/embeddable/public/lib/embeddables/with_subscription.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import * as Rx from 'rxjs';
+import { IEmbeddable, EmbeddableInput, EmbeddableOutput } from './i_embeddable';
+
+export const withEmbeddableSubscription = <
```

Comment:
cc @streamich -- am adding this helper react class.  All my examples ended up with the same subscription code, etc. Lmk if you have any suggestions.

Reply:
This `withEmbeddableSubscription` higher order component seems to be OK. I'm planning to improve Embeddable DX in general for React, so you would not need to write such helpers.